### PR TITLE
Feature/5

### LINKED ISSUE #3: bug: empty array bug
When I use a GET request I get an empty array in insomnia, I think it might have something to do with my code.

### LINKED ISSUE #5: feature: try catch error
feature: try catch error

### PR DESCRIPTION
**Pull request by RomanHyde**
Fixed: empty arrays 
Feature: try catch error

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -9,18 +9,17 @@ const songRouter = require('./routes');
...
-
 const mongoDB = 'mongodb://localhost:27017/my_songs'
 mongoose.connect(mongoDB, {
     useNewUrlParser: true,
     useUnifiedTopology: true 
 });
 
 const db = mongoose.connection;
-db.on('error', console.error.bing(console,'MongoDB connection error:'));
+db.on('error', console.error.bind(console,'MongoDB connection error:'));
```
Comment (nmunro) [inline index.js:19]: Good catch there on that bug 👍

File: models/song.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
+const mongoose = require('mongoose');
+
+const song = new mongoose.Schema({
```
Comment (nmunro) [inline models/song.js]: I would consider using `Song` rather than `song` since models and classes can be non camelCase :)

File: models/song.js [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+        required: true,
+        unique: true
+    },
+    album: {
+        type: String,
+        required: true
+    }
+});
+
+module.exports = mongoose.model('songs', song);
```
Comment (nmunro) [inline models/song.js]: And if you do change it to `Song`, remember to change it here!

File: routes.js [hunk 4]
```diff
@@ -11,17 +11,18 @@ router.get('/', async (req, res) => {
...
+            song: req.body.song,
+            album: req.body.album
         });
-    user.save();
-    res.status(201).send('I created a thing!');
+        await song.save();
+        res.status(201).json(song);
+    } catch (error) {
+        res.status(500).json(error);
+    }
```
Comment (nmunro) [inline routes.js:25]: Good use of error handling and the right status codes (201 for created, 500 for internal server error!)

**Review by nmunro — DISMISSED** [review]
Good work!

File: models/song.js (hunk 2 above)
Comment (RomanHyde) [reply models/song.js]: Working on it!

**Review by nmunro — APPROVED** [review]
Great, thanks for that, it's just good practice :)

**Pull request merged**